JudoPayDotNet/Clients/JudoPayClient.cs

```diff
@@ -16,12 +16,13 @@ namespace JudoPayDotNet.Clients
     /// Base judo pay client that does all the CRUD operations and verifies the existence of errors on responses
     /// </summary>
     public abstract class JudoPayClient
-
     {
         private readonly IClient _client;
+
         private readonly ILog _logger;
-        internal const String SDK_VERSION = "1.3.1";
```

Comment:
I think I put a build step in that dynamically updated this. So that will need updating.